### PR TITLE
CNF-13580: Enable dependabot

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - package-ecosystem: "gomod" # See documentation for possible values
+    directories:
+      - "/"
+      - "addon-tools/*"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 5
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-patch"]
```
[the remark this answers is not in the document]
Comment (nocturnalastro) [reply .github/dependabot.yml:17]: I think its intended to stop too much spam but if we don't care we can remove it :)

**Review by natifridman — APPROVED** [review]
/lgtm

**Comment by AlinaSecret** [comment]
/lgtm
/approve
Thanks :man_dancing:

**Pull request merged**